### ISSUE TITLE
activation for the output of semantic segmentation head

### ISSUE THREAD

**Issue by xiongzubiao** [facebookresearch/detectron2#188]
It seems there is no activation at the output of the semantic segmentation head. Would it be reasonable to have one, at least as an option, since most people may think the semantic segmentation outputs possibility map instead of logits?

**Comment by wuqi930907** [comment]
Yeah, semantic segmentation seems more useful!

**Comment by ppwwyyxx** [comment]
Given it's only one simple call to a softmax function, I don't think it's worth adding an option for it. We'll document its return values more clearly to avoid confusions.
